docs/a320-simvars.md

```diff
@@ -890,6 +890,14 @@
     - Bool
     - Auto brake panel push button for MAX mode is pressed
 
+- A32NX_FM_LS_COURSE
+    - Number
```

Comment:
**suggestion:** `Number<Degrees>` ?

Reply:
... Degrees when valid... -1 when not.... and maybe a special value later when we support backcourse :P